src/Tooltip.js

```diff
@@ -1,64 +1,39 @@
 import React from 'react';
 import t from 'prop-types';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
 
-const toolTipStyles = {
-  background: '#000',
-  borderRadius: '3px',
-  color: '#fff',
-  left: '50%',
-  padding: '5px 10px',
-  position: 'absolute',
-  bottom: '125%',
-  transform: 'translateX(-50%)',
-};
-
-const toolTipBodyBtnStyles = {
-  background: '#352e2e',
-  display: 'inline-block',
-  color: '#eee',
-  margin: '5px 0px',
-  border: '1px solid #352e2e',
-  borderRadius: '5px'
-}
+import { toolTipStyles, toolTipBodyBtnStyles, afterStyles } from './TooltipStyles.js';
 
-const afterStyles = {
-  borderColor: '#000000 transparent transparent transparent',
-  borderStyle: 'solid',
-  borderWidth: '5px 7px 0 7px',
-  bottom: '-5px',
-  height: 0,
-  left: '50%',
-  position: 'absolute',
-  transform: 'translateX(-50%)',
-  width: 0,
+const openEmailClient = email => {
+  console.log('triggered');
+  document.location = `mailto:${email}`;
 };
 
 const Tooltip = ({ emailAddress, className }) => (
-  <div
-    className={className + '__tooltip'}
-    style={toolTipStyles}
-  >
-  <div className={className + '__tooltip-head'}>{ emailAddress }</div>
-  <div className={className + '__tooltip-body'}>
-    <button
-      className={className + '__tooltip--copy-email-address __tooltip-body-btn'}
-      style={toolTipBodyBtnStyles}
-      >Copy Email Address To ClipBoard</button>
-    <button
-      className={className + '__tooltip--compose-default-email __tooltip-body-btn'}
-      style={toolTipBodyBtnStyles}
-      >Compose email in default client</button>
+  <div className={className + '__tooltip'} style={toolTipStyles}>
+    <div className={className + '__tooltip-head'}>{emailAddress}</div>
+    <div className={className + '__tooltip-body'}>
+      <CopyToClipboard text={emailAddress} onCopy={() => alert('copied.')}>
+        <button className={className + '__tooltip--copy-email-address __tooltip-body-btn'} style={toolTipBodyBtnStyles}>
+          Copy Email Address To ClipBoard
+        </button>
+      </CopyToClipboard>
+
+      <button
+        className={className + '__tooltip--compose-default-email __tooltip-body-btn'}
+        style={toolTipBodyBtnStyles}
+        onClick={() => openEmailClient(emailAddress)}
```

Comment:
Do you think we need to make this a `button` with and `onClick`?  We could just make it an `<a href="">`??

Reply:
was being consitent here, as both looks more like a button when look in the design as well. 
Any trade off with using button over anchor here ?

Reply:
Semantics are more important than how the button looks when rendered IMO. If we use a `<a>` we can simply write: `<a href={`mailto:${emailAddress}`}>` and be done with it. No need to click handlers etc as the browser handles `mailto:` prefixes

Reply:
@alexpriceonline I have changed the button to the `<a>`. Can you verify this commit :)